BookingsApi/BookingsApi.Domain/Hearing.cs

```diff
@@ -248,6 +252,7 @@ public Participant AddJudicialOfficeHolder(Person person, HearingRole hearingRol
 
         public void RemoveJudiciaryParticipantByPersonalCode(string judiciaryParticipantPersonalCode)
         {
+            ValidateChangeAllowed();
```

Comment:
just wondering if we need to apply this rule to `UpdateJudiciaryParticipantByPersonalCode` as well, since that will technically allow you to remove a judge by changing the hearing role.

something like this, before UpdateDisplayName:

```
if (participant.HearingRoleCode == JudiciaryParticipantHearingRoleCode.Judge && 
                newHearingRoleCode != JudiciaryParticipantHearingRoleCode.Judge)
            {
                ValidateChangeAllowed();
            }
```

Reply:
also wondering if we should call `ValidateChangeAllowed` from the other participant methods as well (`AddEndpoint`, `AddRepresentative` etc) if they all count as editing a hearing